### PR TITLE
WIP

### PR DESCRIPTION
**Pull request by tadatuta**
Do not merge, PR is just for discussion

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,3 +11,6 @@ npm-debug.log
 .enb/tmp/
 pages/*
 blocks/*
+!blocks/server
```
Comment (tadatuta) [inline .gitignore:14]: not sure if you want server blocks on the same level

File: blocks/server/server.node.js [hunk 2]
```diff
@@ -0,0 +1,1151 @@
...
+    }).on('error', function(err) {
+        console.log('Got error: ' + err.message);
+    });
+}
+
+function renderJson(json) {
+    return '<pre>' + JSON.stringify(json, null, 4) + '</pre>';
+}
+
+var blocks = {
```
Comment (tadatuta) [inline blocks/server/server.node.js:51]: Are you sure you don't want anything better then plain JS to generate BEMJSON?

File: pages/text-only/text-only.bemdecl.js [hunk 3]
```diff
@@ -0,0 +1,678 @@
+exports.blocks = [
```
Comment (tadatuta) [inline pages/text-only/text-only.bemdecl.js:1]: should bemdecl be splited into separate deps files?

**Comment by veged** [comment]
hand merged, thnx!

**Pull request closed without merging**